moabb/datasets/tests.py

```diff
@@ -0,0 +1,38 @@
+'''
+Tests to ensure that datasets download correctly
+'''
+from moabb.datasets.gigadb import GigaDbMI
+from moabb.datasets.alex_mi import AlexMI
+from moabb.datasets.physionet_mi import PhysionetMI
+from moabb.datasets.bnci import BNCI2014001, BNCI2014002, BNCI2014004, BNCI2015001, BNCI2015004
+from moabb.datasets.openvibe_mi import OpenvibeMI
+import unittest
+
+def test_dataset(data):
+    obj = data()
+    obj.subject_list = obj.subject_list[:2]
+    obj.get_data(obj.subject_list)
+
+class Test_Datasets(unittest.TestCase):
+
+    def test_gigadb(self):
+        test_dataset(GigaDbMI)
+
+    def test_bnci(self):
+        test_dataset(BNCI2014001)
+        test_dataset(BNCI2014002)
+        test_dataset(BNCI2014004)
+        test_dataset(BNCI2015001)
+        test_dataset(BNCI2015004)
```

Comment:
could you add tests for all supported datasets in the bnci section

Reply:
as in the non-motor ones too you mean? I didn't work on those at all...I wanted to finish up motor imagery first and then look elsewhere

Reply:
We might just forget about the tests (since the code is here and probably working).

Reply:
The datasets there are all the ones which have been translated into BaseDataset children -- the others have loading functions but nothing else, so it would be a lot of work to get them into the tests.

Reply:
okay, so we can just ignore them for now.